interfaces/fiveg_core_gnb/v0/schema.py

```diff
@@ -0,0 +1,57 @@
+"""This file defines the schemas for the provider and requirer sides of the `fiveg_core_gnb` relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {
+            "tac": 1,
+            "plmns": [
+                {
+                    "mcc": "001",
+                    "mnc": "01",
+                    "sst": 1,
+                    "sd": 1,
+                }
+            ],
+        }
+    RequirerSchema:
+        unit: <empty>
+        app: {
+            "cu_name": "gnb001",
+        }
+"""
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel, Field
+from typing import List
+
+
+class FivegCoreGnbProviderAppData(BaseModel):
+    tac: int = Field(
+        description="Tracking Area Code",
+        examples=[1],
+        ge=1,
+        le=16777215,
+    )
+    plmns: List[dict]
```

Comment:
Is there a reason that this is just a plain list of dictionaries? The text in the README makes it seem like there are fields that should always be included here, so I expected it to be a list of another model that defined those.